src/jit/flowgraph.cpp

```diff
@@ -21523,6 +21527,35 @@ void                Compiler::fgInline()
     BasicBlock* block = fgFirstBB;
     noway_assert(block);
 
+    // Set the root inline context on all statements
+    // 
+    // Note we set ixlCode to nullptr here to preserve existing
+    // behavior, which is to allow one recursive inline.
+    inlExpLst* expDsc = new (this, CMK_Inlining) inlExpLst;
+    expDsc->ixlCode = nullptr;
+    expDsc->ixlParent = nullptr;
+    expDsc->ixlSibling = nullptr;
+    expDsc->ixlChild = nullptr;
+    expDsc->ilOffset = 0;
+
```

Comment:
I'm going to keep things as is for now.. will be coming back to this later when we have a fully descriptive tree and serialization support.